.github/workflows/qpext-docker-publish.yml

```diff
@@ -5,6 +5,7 @@ on:
     # Publish `master` as Docker `latest` image.
     branches:
       - master
```

Comment:
is this still needed?  This PR is for the release branch.

Reply:
It is not needed here. Although, I think, it also doesn't hurt.

If it is noisy, I can remove it.